content/docs/tutorials/data-exploration-in-r/intro-to-r.Rmd

```diff
@@ -254,6 +258,16 @@ columns <- c('retail_and_recreation', 'grocery_and_pharmacy', 'parks', 'transit_
 mobility <- mobility %>% mutate(avg_mobility2 = rowMeans(select(., columns), na.rm =TRUE))
 
 
+```
+
+```{r, webex.hide = 'Optional solution Exercise 7', eval = FALSE}
+# solution (efficient, but prone to couting error) (create an array first, then rowMeans of array)
+
+avg_mobility_array <- mobility[c(6,7,8,9,10,11)]
```

Comment:
Same issue here: referring to columns directly is error-prone. Maybe point it out here as well?